### PR TITLE
Get everything ready to release to hackage

### PR DESCRIPTION
**Pull request by mjgpy3**
This includes
 - Adding `CI` and `Release` workflows
 - Adding changelogs and readmes
 - Updating `package.yaml`s

### CODE REVIEW

**Comment by mjgpy3** [comment]
CircleCI's failing, expectedly.

File: aws-xray-client-persistent/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# aws-xray-client-persistent
+
+A Haskell client for integrating AWS X-Ray with [Persistent](https://hackage.haskell.org/package/persistent).
+
+To use this, you may want choose to define a helper to annotate traces
+
+``` haskell
```
Comment (pbrisbin) [inline aws-xray-client-persistent/README.md]:
```suggestion
```hs
```

File: aws-xray-client-persistent/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          sendTrace = atomicallyAddVaultDataSubsegment vaultData
+          stdGenIORef = xrayVaultDataStdGen vaultData
+          subsegmentName = name <> " query"
+        run . runSqlConn action =<< liftIO
+          (xraySqlBackend sendTrace stdGenIORef subsegmentName backend)
+```
+
+Then you can use this in your `runDB` definition to trace DB calls
+
+``` haskell
```
Comment (pbrisbin) [inline aws-xray-client-persistent/README.md]:
```suggestion
```hs
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  push:
+    branches: main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: freckle/stack-cache-action@v1.0.1
+      - uses: freckle/stack-action@v2
```
Comment (pbrisbin) [inline .github/workflows/ci.yml:14]: This always makes me smile :heart_eyes:

**Review by pbrisbin — APPROVED** [review]
Awesome! I just nit-picked the markdown with how we typically do that.

I can tell Circle to stop building. We'll want to make the Action a Required Status after merge.

File: .github/workflows/ci.yml (hunk 3 above)
Comment (mjgpy3) [reply .github/workflows/ci.yml:14]: @pbrisbin so clean, right?

**Pull request merged**